components/tools/OmeroPy/src/omero/plugins/admin.py

```diff
@@ -1371,6 +1375,10 @@ def open_config(self, unused):
 
         return config
 
+    def die_on_ro(self, config):
```

Comment:
Note: Copied from `prefs.py` so that the `@with_rw_config` decorator can be used.